---
bool is_field_analyzed
